src/model/metadata/test/metadataApiDataService.test.js

```diff
@@ -0,0 +1,18 @@
+import { enableFetchMocks } from "jest-fetch-mock";
+import { describe } from "jest-circus";
+import MetadataApiDataService from "../services/metadataApiDataService";
+import config from "../../../config";
+
+enableFetchMocks();
+
+describe("Metadata api data service", () => {
+  it("fetchCensusMetadata function", async () => {
+    fetch.mockResponseOnce(JSON.stringify([{ test: "test" }]));
+
+    const dataService = new MetadataApiDataService();
+    const json = await dataService.fetchCensusMetadata();
+    expect(fetch).toHaveBeenCalledWith(`${config.api.baseUrl}${config.api.censusMetadataEndpoint}`);
+    expect(Array.isArray(json)).toEqual(true);
```

Comment:
do we need to check type here, as we're checking for equality with an array on the next line?

Reply:
good point, will remove